modules/op-release-notes-1-3.adoc

```diff
@@ -0,0 +1,153 @@
+
+* When a pipeline accepts an optional workspace and gives it to a `PipelineTask`, the `PipelineRun` stalls if the workspace is not provided.
+
+* To use the Buildah cluster task in a disconnected environment, ensure that the Dockerfile uses an internal image stream as the base image, and then use it in the same manner as any S2I cluster task.
+
+
+[id="fixed-issues-1-3_{context}"]
+== Fixed issues
+
+* Extensions added by a CEL Interceptor are passed on to webhook interceptors by adding the `Extensions` field within the event body.
+
+* The activity timeout for log readers is now configurable using the `LogOptions` field. However, the default behavior of timeout in 10 seconds is retained.
+
+* The `log` command ignores the `--follow` flag when a task run or pipeline run is complete, and reads available logs instead of live logs.
+
+* References to the following Tekton resources: `EventListener`, `TriggerBinding`, `ClusterTriggerBinding`, `Condition`, and `TriggerTemplate` are now standardized and made consistent across all user-facing messages in `tkn` commands.
+
+* Previously, if you started a canceled task run or pipeline run with the `--use-taskrun <canceled-task-run-name>`, `--use-pipelinerun <canceled-pipeline-run-name>` or `--last` flags, the new run would be canceled. This bug has now been fixed.
+
+* The `tkn pr desc` command  is now enhanced to ensure that it does not fail in case of pipeline runs with conditions.
+
+* When you delete a task run using `tkn tr delete` with the `--task` option, and a cluster task exists with the same name, the task runs for the cluster task also gets deleted. As a workaround, filter the task runs by using the `TaskRefKind` field.
+
+* The `tkn triggertemplate describe` command would display only part of the `apiVersion` in the output. For example, only `triggers.tekton.dev` was displayed instead of `triggers.tekton.dev/v1alpha1`. This bug has been fixed.
+
+* The webhook, under certain conditions, would fail to acquire a lease and not function correctly. This bug has been fixed.
+
+* Pipelines with `When` expressions created in v0.16.3 can now be run in v0.17.1 and later. After an upgrade, you do not need to reapply pipeline definitions created in previous versions because both the uppercase and lowercase first letters for the annotations are now supported.
+
+* By default, the `leader-election-ha` is now enabled for high availability. When the controller flag `disable-ha` is set to `true`, it disables high availability support.
+
+* Issues with duplicate cloud events are now fixed. Cloud events are now sent only when a condition changes the state, reason, or message.
+
+* When a service account name is missing from a `PipelineRun` or `TaskRun` spec the controller uses the service account name from the `config-defaults` config map. If it is missing in the `config-defaults` config map too, the controller now sets it to `default` in the spec.
```

Comment:
spec, the controller

If the service account name is also missing in the the `config-defaults` config map